dashboard/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
While all of this is useful, it's a README for the react npm application template generator rather than for the Pbench dashboard. Can we rename this so we can work towards a dashboard `README.md`?

Reply:
:100:  agree with you on this.
I'll start looking into updating this on further releases.
Creating a story to track this effort real quick ...

Reply:
Here it is..
https://issues.redhat.com/browse/PBENCH-96